.github/workflows/fsync.yaml

```diff
@@ -0,0 +1,19 @@
+name: fsync
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '*/30 * * * *' # Runs every 30 minutes
```

Comment:
we can do daily or even weekly prob instead of 30 minutes, not going to change every 30 minutes